bentoml/cli/__init__.py

```diff
@@ -346,6 +346,7 @@ def serve_gunicorn(
                     port=port,
                     num_of_workers=microbatch_workers,
                     prometheus_lock=prometheus_lock,
+                    model_server_workers=workers or 3,
```

Comment:
3 is actually not the default value, I think it should use this function instead to get the default `workers` value:
https://github.com/bentoml/BentoML/blob/master/bentoml/server/utils.py#L29
https://github.com/bentoml/BentoML/blob/master/bentoml/server/gunicorn_config.py#L7

Reply:
@parano updated